openbb_platform/providers/yfinance/openbb_yfinance/models/company_news.py

```diff
@@ -33,7 +33,7 @@ class YFinanceCompanyNewsData(CompanyNewsData):
     uuid: str = Field(description="Unique identifier for the news article")
     publisher: str = Field(description="Publisher of the news article")
     type: str = Field(description="Type of the news article")
-    thumbnail: Optional[List] = Field(
+    thumbnail: Optional[List[Dict[str, Any]]] = Field(
         default=None, description="Thumbnail related data to the ticker news article."
```

Comment:
This field needs standardization between all providers. We have:

- "image"
- "image_url"
- "thumbnail"
- "images"

Reply:
@hjoaquim has a ticket that deals with this. @deeleeramone maybe also open an issue on our bugs ticket.

Reply:
Agreed. But please, let's open a new issue and keep this PR within its scope!!